### PR TITLE
chore: removing v1 code from the tests

### PR DESCRIPTION
**Pull request by nicolasburtey**

### CODE REVIEW

File: test/integration/services/ledger/transaction.spec.ts [hunk 1]
```diff
@@ -105,7 +105,7 @@ describe("receipt", () => {
       description: "transaction test",
       payeeUser: walletBTC,
       metadata: { type: "invoice", pending: false },
-      sats: 1000,
+      sats: 1000 as Satoshis,
```
Comment (dolcalmi) [inline test/integration/services/ledger/transaction.spec.ts:108]: please use toSats (applies to other changes in this file)

**Review by dolcalmi — CHANGES_REQUESTED** [review]
looks good, just a couple of comments. @vindard please check before merge (although most of the changes are in the tests we need to be careful, specially with the projections)

File: src/graphql/admin/root/query/transaction-by-id.ts [hunk 2]
```diff
@@ -13,7 +12,6 @@ const TransactionByIdQuery = GT.Field({
     if (id instanceof Error) throw id
 
     const ledgerTx = await Wallets.getTransactionById(id)
-    if (ledgerTx instanceof CouldNotFindTransactionError) return null
```
[the remark this answers is not in the document]
Comment (nicolasburtey) [reply src/graphql/admin/root/query/transaction-by-id.ts:16]: good catch. I didn't meant to include it, but was pulled over from a rebase. removed.

**Review by vindard — COMMENTED** [review]
Left some comments about variable naming in the tests to be more specific. I'm ok to approve otherwise once [this change](https://github.com/GaloyMoney/galoy/pull/881/files#r780127012) is made

File: test/integration/02-user-wallet/02-is-user-active.spec.ts [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
 
-import { getAndCreateUserWallet } from "test/helpers"
+import { createUserWallet, getUserIdByTestUserIndex } from "test/helpers"
 
 describe("getRecentlyActiveAccounts", () => {
   it("returns active users according to volume", async () => {
-    await getAndCreateUserWallet(0)
+    await createUserWallet(0)
+
+    const user0 = await getUserIdByTestUserIndex(0)
```
[the remark this answers is not in the document]
Comment (nicolasburtey) [reply test/integration/02-user-wallet/02-is-user-active.spec.ts:11]: done

File: test/integration/02-user-wallet/02-receive-lightning.spec.ts [hunk 4]
```diff
@@ -3,28 +3,29 @@ import * as Wallets from "@app/wallets"
...
+  createUserWallet,
   getBTCBalance,
+  getDefaultWalletIdByTestUserIndex,
   getHash,
   lndOutside1,
   pay,
 } from "test/helpers"
 
-let userWallet1
+let wallet1: WalletId
```
[the remark this answers is not in the document]
Comment (nicolasburtey) [reply test/integration/02-user-wallet/02-receive-lightning.spec.ts]: done

File: test/integration/02-user-wallet/02-index.spec.ts [hunk 5]
```diff
@@ -1,42 +1,69 @@
...
+    createMandatoryUsers()
+
+    await createUserWallet(0)
+    await createUserWallet(2)
+
+    userType0 = await getUserTypeByTestUserIndex(0)
+    userType2 = await getUserTypeByTestUserIndex(2)
+
+    wallet0 = await getDefaultWalletIdByTestUserIndex(0)
+    account0 = await getAccountIdByTestUserIndex(0)
```
[the remark this answers is not in the document]
Comment (nicolasburtey) [reply test/integration/02-user-wallet/02-index.spec.ts]: I suggest to do that in another PR

File: test/integration/02-user-wallet/02-index.spec.ts [hunk 6]
```diff
@@ -59,54 +86,80 @@ describe("UserWallet", () => {
...
 
-    expect(userWallet2.user.oldEnoughForWithdrawal).toBeTruthy()
+    expect(userType2.oldEnoughForWithdrawal).toBeTruthy()
   })
 
   describe("setUsername", () => {
     it("does not set username if length is less than 3", async () => {
-      await expect(userWallet0.setUsername({ username: "ab" })).rejects.toThrow()
+      await expect(setUsername({ username: "ab", id: account0 })).resolves.toBeInstanceOf(
+        ValidationError,
```
[the remark this answers is not in the document]
Comment (nicolasburtey) [reply test/integration/02-user-wallet/02-index.spec.ts]: I suggest to do that in another PR

File: test/integration/02-user-wallet/02-index.spec.ts (hunk 5 above)
[the remark this answers is not in the document]
Comment (nicolasburtey) [reply test/integration/02-user-wallet/02-index.spec.ts]: done

**Comment by nicolasburtey** [comment]
> Left some comments about variable naming in the tests to be more specific. I'm ok to approve otherwise once [this change](https://github.com/GaloyMoney/galoy/pull/881/files#r780127012) is made

the link for this change doesn't work

**Review by vindard — APPROVED** [review]

**Comment by vindard** [comment]
> @vindard please check before merge (although most of the changes are in the tests we need to be careful, specially with the projections)

@dolcalmi yup checked the projection against the translate function, and anything else in non-test code. Looked ok so far.

**Pull request merged**